base/inference.jl

```diff
@@ -3359,7 +3360,8 @@ function is_pure_builtin(f::ANY)
              f === Intrinsics.checked_srem_int ||
              f === Intrinsics.checked_urem_int ||
              f === Intrinsics.check_top_bit ||
-             f === Intrinsics.sqrt_llvm)
+             f === Intrinsics.sqrt_llvm ||
+             f === Intrinsics.cglobal)  # cglobal throws an error for symbol-not-found
```

Comment:
I think we should just fix that code. I've never been super happy with it throwing an exception during every normal startup anyways.